loops_lab.js

```diff
@@ -0,0 +1,202 @@
+ let num = 10;
+
+ while(num >=1 ){
+   console.log(num)
+   num -=1
+ }
+
+ console.log("-------")
+
+ for (let num = 20; num >= 1; num -= 1 ){
+   console.log(num)
+ }
+
+console.log("-------")
+// 2. Write a `while` loop and a `for` loop that takes the variable "max", and iterates over all numbers from 0 to "num".
+// For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
+
+for ( let max = 0 ; max <=  14 ; max +=1) {
+  if (max % 2 !== 0) {
+  console.log(max + " is odd")
+} else {
+    console.log(max + " is even")
+  }
+}
+
+console.log("------")
+
+// while loop
+let max = 0 ;
+while( max <= 15){
+    if (max % 2){
+      console.log(max + " is odd")
+    } else if ( max % 2 === 0) {
+        console.log(max + " is even")
+        }
+    max ++
+}
+
+// 3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num".
+// For each iteration of the loop, it will multiply the number by 9 and log the result (e.g. "2 * 9 = 18").
+//   _Bonus_ think of another way to solve it.
+//     <details>
+//       <summary>
+//         Hint
+//       </summary>
+//       Find the final number and increment the loop by 9.
+//     </details>
+
+let num3 =0;
+while (num3 <= 20) {
+    console.log(num3 * 9)
+ num3 ++
+}
+
+console.log("-------")
+
+for(let num3 = 0 ;num3 <= 25; num3 ++) {
+    console.log(num3 * 9)
+}
+
+console.log("-------")
+//4. Write code that prints/logs all the numbers that end in a 5 from 1 to 100, exclusive.
+     for (let count5 =1; count5 <=100; count5 += 1){
+          if ((count5 % 5 === 0) && (count5 % 10)){
+              console.log(count5)
+            }
+     }
+console.log("-------")
+//5.Without running/executing your code, how many times will the loop below run? Explain why.
+
+// let i = 5;
+//
+// while (i > 3) {
+//     i += 1
+// }
+
+// Your explanation here
+   // the loop will perform only twice since i is set at the limit of 5 and the loop starts at 3
+
+
+// 6. Write a loop that uses console.log to log all the numbers from 1 to 100, with two exceptions. For numbers divisible by 3, log "Fizz" instead of the number, and for numbers divisible by 5 (and not 3), log "Buzz" instead.
+// console.log( "---------")
+
+for (let i = 0; i <= 100; i ++){
+  if ( i % 3 === 0){
+    console.log("Fizz")
+  } else if ( i % 5 === 0 && i % 3 ){
+      console.log( "Buzz")
+  } else {
+      console.log(i)
+  }
+}
+
+console.log("-------")
+// 7. Modify your program to log "FizzBuzz", for numbers that are divisible by both 3 and 5 (still log "Fizz" or "Buzz" for numbers divisible by only one of those).
+//console.log( "---------")
+
+for ( let i = 0; i <= 100; i ++){
+  if ( i % 3 === 0){
+    console.log("Fizz")
+  } else if ( i % 5 === 0){
+      console.log( "Buzz")
+  } else {
+      console.log(i)
+  }
+  if ( (i % 3 ===0 ) && (i % 5 ===0)){
```

Comment:
When I ran the code, it was giving both Fizz and FizzBuzz for numbers such 30 or 75. To avoid that issue, move "if ( (i % 3 ===0 ) && (i % 5 ===0))" to the top so that the code stops running after satisfying FizzBuzz.